src/main/java/org/folio/marc4ld/service/marc2ld/field/FieldControllerImpl.java

```diff
@@ -0,0 +1,79 @@
+package org.folio.marc4ld.service.marc2ld.field;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.collections4.CollectionUtils;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.fingerprint.service.FingerprintHashService;
+import org.folio.marc4ld.dto.MarcData;
+import org.folio.marc4ld.service.marc2ld.Marc2ldFieldRule;
+import org.folio.marc4ld.service.marc2ld.mapper.Marc2ldMapper;
+import org.folio.marc4ld.service.marc2ld.mapper.Marc2ldMapperController;
+import org.folio.marc4ld.service.marc2ld.mapper.mapper.MapperHelper;
+import org.folio.marc4ld.service.marc2ld.relation.RelationProvider;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FieldControllerImpl implements FieldController {
+
+  private final RelationProvider relationProvider;
+  private final Marc2ldMapperController marc2ldMapperController;
+  private final FingerprintHashService hashService;
+  private final MapperHelper mapperHelper;
+
+  @Override
+  public void handleField(
+    Resource parent,
+    DataField dataField,
+    List<ControlField> controlFields,
+    Marc2ldFieldRule fieldRule
+  ) {
+    var fieldMapper = getMapper(dataField, controlFields, fieldRule);
+    var mappedResources = fieldMapper.createResources(parent);
+
+    mappedResources
+      .forEach(resource -> additionalMapping(resource, fieldRule, dataField, controlFields));
```

Comment:
what's the point of a new line here?
(general comment for all similar cases)

Reply:
discussed